libmscore/paste.cpp

```diff
@@ -324,9 +324,10 @@ bool Score::pasteStaff(XmlReader& e, Segment* dst, int dstStaff)
 
                               // be sure to paste the element in the destination track;
                               // setting track needs to be repeated, as it might have been overwritten by el->read()
-                              el->setTrack(e.track());
+                              // preserve *voice* from source, though
+                              el->setTrack(e.track() + el->voice());
```

Comment:
I'm inclined to agree, but I figured the change was made for a reason.  I could also see taking (e.track() / VOICES) * VOICES.  @wschweer, is there a reason to prefer one of these?